R/annotation-data.R

```diff
@@ -13,10 +13,32 @@
 #'
 #' @examples
 #' \dontrun{
-#' promoterAnnotation <- preparePromoterAnnotation(txdb,
+#' promoterAnnotation <- preparePromoterAnnotation(file,
 #'                                                 species = 'Homo_sapiens')
 #' }
-preparePromoterAnnotation <- function(txdb, species) {
+preparePromoterAnnotation <- function(file, species) {
```

Comment:
users could also load the txdb object in R and then run this function (without providing a file, just by providing the txdb object). Do you want to add that option, mabye by having 2 arguments function(txdb, file) only one of which has to be provided?